### PR TITLE
Added index.html file

### PR DESCRIPTION
**Pull request by Nivet2006**
@dev-minds0987 
I want you to merge this pull request. If you think there is no error while debugging. 
If you have found an error, please let me know.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,487 @@
+<!DOCTYPE html>
+<html lang="en" >
+<head>
+<meta charset="UTF-8">
```
Comment (dev-minds0987) [inline index.html:4]: Hello @nived-dev , I am very impressed that you have submitted the code within 6 days but the favicon image cannot be seen **try checking the code again.** 
Anyway considering your work, I am considering this merge request

**Review by dev-minds0987 — CHANGES_REQUESTED** [review]
Check the code tomorrow and get back to me ASAP

**Comment by dev-minds0987** [comment]
@nived-dev Pull requested will be merged but please make the changes and get back to me

**Comment by Nivet2006** [comment]
Branch deleted from github.com/minds0987.
For more info: https://github.com/minds0987/calculaty/pull/1
OR
mailto:minds0987@gmail.com

**Pull request closed without merging**